msf/preflight.go

```diff
@@ -273,11 +271,8 @@ func _preflight(readerFn func(callback gcode.LineCallback) error, palette *Palet
 			results.totalLayers++
 			results.layerTopZs = append(results.layerTopZs, 0)
 			results.layerThicknesses = append(results.layerThicknesses, 0)
-			results.layerObjectStarts = append(results.layerObjectStarts, 0)
-			results.layerObjectEnds = append(results.layerObjectEnds, 0)
-			if lastTurnOnFanCommand >= 0 && lastTurnOnFanCommand == lineNumber-1 {
-				results.lastFanOnLineBeforeLayerChange = lastTurnOnFanCommand
-
+			if lastFanCommandLine >= 0 && lastFanCommandLine == lineNumber-1 {
+				results.lastFanOnLineBeforeLayerChange = lastFanCommandLine
```

Comment:
Similarly, can we rename this variable for accuracy?

```suggestion
				results.lastFanCommandLineBeforeLayerChange = lastFanCommandLine
```